import_export/templates/admin/import_export/import.html

```diff
@@ -29,7 +29,12 @@
 
       <p>
         {% trans "This importer will import the following fields: " %}
-        <code>{{ fields|join:", " }}</code>
+        <dl>
```

Comment:
can we wrap this around an `<% if %>` so that the existing formatting is preserved if there is only one `Resource`.  Users may have applied their own css formatting and the new version may break this.  Hence if there is only one resource then the original formatting will be applied.

Reply:
OK, done that. Although now it would need to be styled twice, if someone would like to have both single resource and multiple resources in his/hers project.

Reply:
Good point.  I think we can live with that.  I think the key thing is that it remains consistent for those not wanting to use multiple classes (which will be the majority).